src/api/app/assets/javascripts/webui/request_show_redesign/add_review.js

```diff
@@ -0,0 +1,24 @@
+function handleReviewerCollapsibleForm() {
+  $('#add-review-dropdown-component .dropdown-item').on('click', function() {
+    var review = $(this);
+
+    $('#review-form-collapse h5 i').html(review.html());
+    $('#review_id').val(review.data('review'));
+  });
+
+  $('#add-review-dropdown-component').on('shown.bs.dropdown', function () {
+    $('#review-form-collapse').collapse('hide');
+  });
+
+  $('.toggle-review-form').on('click', function() {
+    $('#review-form-collapse h5 i').html(this.dataset.reviewerIcon);
+    $('#review_id').val(this.dataset.review);
+  });
+
+  $(document).click(function(e) {
+    var reviewCollapsible = document.getElementById('review-form-collapse');
+    if (!reviewCollapsible.contains(e.target)) {
+        $('.collapse').collapse('hide');
```

Comment:
This is affecting all collapse elements on the page where this code is executed. Only the `review-form-collapse` element should be hidden.

For example, comments are affected, preventing users from writing comments:
![comments-collapsing](https://user-images.githubusercontent.com/1102934/184622117-18b19105-60a1-49ef-a7de-4e5ebc862c9f.gif)

Reply:
Fix pushed.